_posts/2023-05-30-settings-page.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Using the improved settings page in Cryostat 2.3.0
+date: 2023-05-30
+synopsis: Read about the new configurable settings in Cryostat 2.3.0.
+author: Max Cao
+---
+
+#### Table of Contents
+* auto-gen TOC:
+{:toc}
+<hr>
+
+Hello readers!
+
+Cryostat 2.3 introduces an improved settings page that offers a wide range of configuration options to optimize the application and tailor it to your specific needs. Whether you need to adjust the theme, language, date locale, or other settings, the settings page provides a user-friendly interface that makes it easy to fine-tune Cryostat to your liking.
+
+### [Settings Page](#settings-page)
+Click the cog icon on the upper right corner of the web interface to take you to the General tab of the settings page.
+
+#### [General](#general)
+![Alt text](/images/blog/settings-page-general.png "General settings tab")
+The **General** settings tab is where you can adjust various general settings to tailor your experience with the application. You can select a preferred theme, the date locale, and the current timezone.
+##### [Theme](#theme)
+Allows you to configure the theme of the Cryostat UI. Currently Cryostat only supports the Light and Dark theme. The web page defaults to `Auto` which defaults to the current theme of the browser you are working with.
+##### [Date & Time](#date-time)
+Allows you to configure the current date locale, as well as the current timezone for more accurate analysis on any time-dependent data.
+
+#### [Connectivity](#connectivity)
+![Alt text](/images/blog/settings-page-connectivity.png "Connectivity settings tab")
+The **Connectivity** settings tab allows you to configure settings related to the Cryostat web client and its WebSocket connection to the Cryostat server.
+##### [WebSocket Connection Debounce](#websocket-connection-debounce)
+This setting allows you to configure the retry interval for establishing WebSocket connections. Increase this time if the web-interface repeatedly displays WebSocket connection/disconnection messages. Decrease this time if the web-interface takes a long time to populate on startup.
+##### [Auto-Refresh](#auto-refresh)
+Sets the refresh period for content views. Views normally update dynamically via WebSocket notifications, so this should not be needed unless WebSockets are not working.
+
+#### [Notifications & Messages](#notifications-messages)
+![Alt text](/images/blog/settings-page-notifications&messages.png "Notifications & Messages settings tab")
+The **Notifications & Messages** tab allows you to configure settings related to the dynamic status notifications that pop up in the web interface.
+##### [Notifications](#notifications)
+Here you can enable/disable notifications (enabled by default). By clicking `Show more` you can filter which type of notifications you want to receive. You may also control the maximum number of notification alerts that appear at once.
```

Comment:
`Show more` should be bolded instead since it is not a Cryostat terminology but rather a UI element text.